pandas/tests/groupby/test_apply.py

```diff
@@ -1583,3 +1583,110 @@ def f_4(grp):
     e.loc["Pony"] = np.nan
     e.name = None
     tm.assert_series_equal(result, e)
+
+
+# GH58291
+def test_apply_frame_not_as_index_returns_single_index():
+    df = DataFrame(
```

Comment:
Can you move inside the function to agree with the style of the other tests.

```
def test_...():
    # GH#58291
```

Reply:
resolved